ReleaseNotes.md

```diff
@@ -22,6 +22,7 @@ Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate t
      - Increase Timeout for OMS Homing service API calls from 30s to 60s
    - Fix for https://github.com/Azure/AKS/issues/2457
    - In replicaset, tailing of the mdsd.err log file to agent telemetry
+   - Daemonset memory requests and limits increased by 25% due to additional hop with MDSD
```

Comment:
I looked at the xls and dont see where you are seeing 25% correlation. I see some have doubled in memory usage.

Reply:
> I looked at the xls and dont see where you are seeing 25% correlation. I see some have doubled in memory usage.
> yes, there is no consistent pattern and some cases, I see 20% and some cases I see 40% , and some cases it got doubled.  in one of the sample clusters, i see better memory usage with mdsd than the previous agent. It could be related to our telemetry sample interval and current thinking is increase 25% and watch the release, and see if we need to adjust further.